configure

```diff
@@ -4876,7 +4876,7 @@ echo "$_libunity"
 #
 find_freetype() {
         # Wrapper function which tries to find freetype
-        # either by calling freetype-config or by using
+        # either by callimg freetype-config or by using
```

Comment:
Why that change? Wasn’t `calling` correct?

Reply:
Grr, of course it was correct.
I thought i was adding this change to my other patch while it was being merged and ended up creating a new PR but with the old data.
Sorry